E       RuntimeError: Database access not allowed, use the "django_db" mark, or the "db" or "transactional_db" fixtures to enable it.

c:\users\kevin\appdata\local\programs\python\python37\lib\site-packages\django\db\backends\base\base.py:233: RuntimeError